system_metrics_collector/launch/talker_listener_example.launch.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
launch/*.py -> examples/launch/*.py ?

Reply:
Resolved per in person comments / pair programming.